source/main.bmx

```diff
@@ -1572,6 +1572,24 @@ endrem
 		'update regardless of enabled or not
 		DebugScreen.UpdateSystem()
 
+
+		'handle (temporary) fast forward here - saving game not available in screen
+		Local fastForwardScreen:TDebugScreenPage_Misc = TDebugScreenPage_Misc.GetInstance()
+		If fastForwardScreen.FastForward_Active and fastForwardScreen.FastForward_TargetTime < GetWorldTime().GetTimeGone()
+			fastForwardScreen.Dev_StopFastForwardToTime()
+		EndIf
+		'continuous fast forward: save game and go to the end of the next day
+		If fastForwardScreen.FastForward_Continuous_Active and fastForwardScreen.FastForward_TargetTime < GetWorldTime().GetTimeGone()
+			Local daysRun:Int = GetWorldTime().GetDaysRun() + 1
+			If daysRun mod 5 = 0
+				Local savegameName:String = "savegames/AI-day-" + StringHelper.RSetChar(daysRun,3,"0")
+				Local savegameURI:String = savegameName + "." + GameConfig.GetSavegameExtension()
+				TSaveGame.Save(savegameURI, savegameName, False)
```

Comment:
Vielleicht einfach ueber TGameConfig  eine Variable bereitstellen die sagt "savegameCreationRequested:Int".

Und dann in der Updateloop am _ENDE_ speichern, falls die Variable True ist.
F5 usw wuerden dann auch nicht direkt speichern ... sondern nur die Variable setzen.

Haette den Vorteil, dass nicht in der Mitte eines Updatevorgangs gespeichert wuerde.